version.props

```diff
@@ -31,6 +31,7 @@
     <VersionSuffix>$(PreReleaseLabel).$(_BuildNumberSuffix)</VersionSuffix>
 
     <ExperimentalVersionPrefix>0.1.$(PatchVersion)</ExperimentalVersionPrefix>
+    <JSInteropVersionPrefix>0.8.$(PatchVersion)</JSInteropVersionPrefix>
```

Comment:
We can give it any version number we want. I think it would make sense for 1.0.0 to be the first stable version we ship, so it would follow that we could version it as 1.0.0-preview-buildnumber until then.

But if there's any practical reason why it's easier to give it any other version number (e.g., 3.0.0 to match ASP.NET Core) that would be totally fine too.

Do you have a preference?

Reply:
3.0.0 would certainly be less work to explain and manage.